### DISCUSSION TITLE
Migrating from CC2531 to Zzh: do I have to delete my devices from the database?

### DISCUSSION THREAD

**Discussion by miguelarios** [Koenkk/zigbee2mqtt#7566] [General]
I have read the documentation that if I change hardware devices that are different base hardware I need to repair my devices. What I’m curious is since I run in a container do I need to just start with a fresh install of the container and delete my devices database?

Do I need to do an unpairing or sorts?

What happens if all I change is the configuration.yaml file and don’t delete the devices in the database? Does this mess with the migration?

**Comment by Koenkk** [comment]
See https://www.zigbee2mqtt.io/information/FAQ.html#how-do-i-migrate-from-a-cc2531-to-a-more-powerful-coordinator-eg-zzh

**Reply by miguelarios** [reply]
Sweet! Thank you! I'll do this! So I take it I don't have to unpair any devices?

**Reply by miguelarios** [reply]
Btw I have quite a few hue bulbs for those do I have to do the touchlink repairing to pair with the new coordinator?

**Comment by KaHeMu** [comment]
I would like to migrate my >50 devices from a CC2531 coordinator (FW 20201128 source routing) to Slaesh's CC2652 stick. I read  [ https://www.zigbee2mqtt.io/information/FAQ.html#how-do-i-migrate-from-a-cc2531-to-a-more-powerful-coordinator-eg-zzh] but I'm still unclear if I have to re-pair all devices or not since it says: "Requires repairing ... Except when switching between adapters with the following chips: **CC2652**, CC1352, **CC253*** (only when running Zigbee 3.0 firmware)"
Any chance to avoid re-pairing my >50 devices?  Thanks!

**Reply by Koenkk** [reply]
Yes, since you are on the 1.2 CC2531 firmware.
